recipe/meta.yaml

```diff
@@ -25,9 +25,11 @@ requirements:
 test:
```

Comment:
There are missing packages in `host`:
- Missing wheel package in requirements
- Missing setuptools package in requirements

Reply:
The following dependencies were added to the `host` section:

`- setuptools`
`- wheel`

https://github.com/AnacondaRecipes/coverage-feedstock/blob/update_to_6.1.1/recipe/meta.yaml#L23&L24